src/test/java/org/truemail/ClientTests.java

```diff
@@ -0,0 +1,70 @@
+package org.truemail;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.truemail.client.Configuration;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import static junit.framework.Assert.assertEquals;
+import static org.powermock.api.mockito.PowerMockito.*;
```

Comment:
Please avoid unused import.

Reply:
Actually, it used for methods `when` and `mock`